src/js/components/ExpandingTable.js

```diff
@@ -7,10 +7,42 @@ import Util from "../utils/Util";
 const WHITESPACE = "\u00A0";
 
 class ExpandingTable extends React.Component {
-  constructor() {
-    super(...arguments);
+  constructor(props) {
+    super(props);
 
-    this.state = { expandedRows: {} };
+    let initialState = { expandedRows: {} };
+
+    if (props.expandRowsByDefault) {
+      initialState = props.data.reduce(
+        function(memo, deployment) {
```

Comment:
I am with @nLight here. `deployment` is too coupled to the deployments tab lets stick to something generic like @nLight suggested.